main/src/com/google/refine/util/LocaleUtils.java

```diff
@@ -0,0 +1,26 @@
+package com.google.refine.util;
+
+import java.util.Locale;
+import java.util.Map;
+
+public class LocaleUtils {
+    private final static Map<String, String> languageCodeMap = Map.of(
+            "bn", "ben",
+            "jp", "ja",
+            "zh", "zh_CN",
+            "zh_Hant", "zh_TW"
+    );
```

Comment:
The initialization of `languageCodeMap` is not run before `setLocale` is run, probably because of the specifics of in which order Java executes static initialization.

Instead, I would recommend you initialize `languageCodeMap` to `null`, and properly initialize it (to `Map.of(...)`) the first time it is needed. Alternatively, because this map is rather small and we are not expecting `setLocale` to be run many times, it is probably also fine not to have it as a static variable, but a local one inside of `setLocale()`.